packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -301,7 +301,7 @@ class SemanticsData extends Diagnosticable {
   ///
   /// If this value is non-null, the SemanticsNode must not have any children
   /// as those would be replaced by the semantics nodes of the referenced
-  /// platform view.
+  /// platform view, and other semantic configurations are ignored.
```

Comment:
Can you clarify what you mean here? (Which other semantic configurations?)

Reply:
I have updated the comments to include the things are ignored. Please let me know if you think we can do it a different way.